kraft.yaml

```diff
@@ -1,18 +1,18 @@
 specification: '0.5'
```

Comment:
Shouldn't this also be set to '0.6'?

Reply:
Setting this to '0.6' would require a `specification_v0.6.json` file here: https://github.com/unikraft/kraft/tree/staging/kraft/config

@nderjung Is it ok to keep '0.5' here so that everything can be run and then bump it in a separate commit when the 0.6 specification will be available in the kraft repo?

Reply:
@razvanvirtan has a point. The specification is for kraft and that's still version 0.5.

Reply:
Also, @razvanvirtan , please change `relates` to `related` in the commit message, as @craciunoiuc pointed out.